src/gainmap.c

```diff
@@ -1,7 +1,6 @@
 // Copyright 2023 Google LLC
 // SPDX-License-Identifier: BSD-2-Clause
 
-#include "avif/gainmap.h"
```

Comment:
Nit: Also delete `#include "avif/avif.h"`. See https://github.com/AOMediaCodec/libavif/pull/1589 for including avif/internal.h vs. avif/avif.h.

Note: I don't strictly enforce this policy in the files in tests/gtest/.